src/sentry/api/helpers/group_index/delete.py

```diff
@@ -95,13 +101,24 @@ def delete_group_list(
     # `Group` instances that are pending deletion
     GroupInbox.objects.filter(project_id=project.id, group__id__in=group_ids).delete()
 
-    delete_groups_task.apply_async(
-        kwargs={
-            "object_ids": group_ids,
-            "transaction_id": str(transaction_id),
-            "eventstream_state": eventstream_state,
-        }
-    )
+    if options.get("deletions.groups.use-new-task"):
```

Comment:
Besides tests, this module is the only place where we schedule the group deletion task, thus, we only use the option here.
https://github.com/search?q=repo%3Agetsentry%2Fsentry%20%22from%20sentry.deletions.tasks.groups%22&type=code